compiler/rustc_codegen_llvm/src/llvm/ffi.rs

```diff
@@ -482,6 +483,7 @@ pub struct SanitizerOptions {
     pub sanitize_thread: bool,
     pub sanitize_hwaddress: bool,
     pub sanitize_hwaddress_recover: bool,
+    pub sanitize_memtag: bool,
```

Comment:
The `SanitizerOptions` should match `LLVMRustSanitizerOptions` defined in `PassWrapper.cpp` exactly.

The field `sanitize_memtag` is not necessary?

Reply:
Ah that is the case, thanks!